compiler/fir/fir2ir/tests-gen/org/jetbrains/kotlin/test/runners/codegen/FirBlackBoxCodegenTestGenerated.java

```diff
@@ -6,10 +6,10 @@
 package org.jetbrains.kotlin.test.runners.codegen;
 
 import com.intellij.testFramework.TestDataPath;
-import org.jetbrains.kotlin.test.util.KtTestUtil;
```

Comment:
Please, don't modify the order of imports in the auto-generated files; that causes some build checks to fail. Make sure that you don't have "Before Commit" actions enabled.